base/person.rb

```diff
@@ -21,6 +22,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    rental.person = self unless rental.person.include?(self)
+    @rentals << rental
+  end
```

Comment:
- [ ] Similarly, we want to be able to use the person and date objects are arguments when creating a new _rental_. We can achieve this by modifying our code to look like the following example below:

```ruby
class Person
  attr_accessor :name, :age, :rentals
  .
  .
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
  .
  .
end
```